File: data/src/main/java/se/tink/repository/service/UserConfigurationService.java
package se.tink.repository.service;

import se.tink.core.models.user.UserConfiguration;
import se.tink.repository.ObjectChangeObserver;

public interface UserConfigurationService extends TinkService {

	void subscribe(ObjectChangeObserver<UserConfiguration> listener);

	void unsubscribe(ObjectChangeObserver<UserConfiguration> listener);

	void refreshUserConfiguration();
}

File: data-commons/src/main/java/se/tink/android/repository/service/UserConfigurationService.kt
package se.tink.android.repository.service;

import com.tink.model.user.UserProfile;
import com.tink.service.observer.ChangeObserver;

@Deprecated
public interface UserConfigurationService {

	void subscribe(ChangeObserver<UserProfile> listener);

	void unsubscribe(ChangeObserver<UserProfile> listener);

	void refreshUserConfiguration();
}
